Mage.Tests/src/test/java/org/mage/test/cards/single/ncc/ThreefoldSignalTest.java

```diff
@@ -173,4 +174,55 @@ public void fusedSplitCardTriggers() {
         assertPermanentCount(playerA, "Bird Token", 4 + 4);
         assertHandCount(playerA, 4 + (4+4));
     }
+
+    /**
+     * Test that casting a split card with fuse triggers if both halves together have 3 colors
+     */
+    @Test
+    public void fusedSplitCardTriggers2() {
+        // {3}{B}{G} / {R}{G}
+        // Flesh: Exile target creature card from a graveyard.
+        //        Put X +1/+1 counters on target creature, where X is the power of the card you exiled.
+        // Blood: Target creature you control deals damage equal to its power to any target.
+        // Fuse
+        String fleshBlood = "Flesh // Blood";
+        addCard(Zone.HAND, playerA, fleshBlood);
+
+        addCard(Zone.BATTLEFIELD, playerA, threefoldSignal);
+        addCard(Zone.BATTLEFIELD, playerA, "Swamp", 1);
+        addCard(Zone.BATTLEFIELD, playerA, "Forest", 2);
+        addCard(Zone.BATTLEFIELD, playerA, "Mountain", 1+3 + 3); // For red + generic costs and to have enough for replicate
+
+        // All are 2/2
+        String lion = "Silvercoat Lion";
+        addCard(Zone.BATTLEFIELD, playerA, lion);
+        String griffin = "Abbey Griffin";
+        addCard(Zone.GRAVEYARD, playerA, griffin);
+        String centaur = "Accursed Centaur";
+        addCard(Zone.GRAVEYARD, playerA, centaur);
+
+        setStrictChooseMode(true);
+        castSpell(1, PhaseStep.PRECOMBAT_MAIN, playerA, "fused Flesh // Blood");
+        setChoice(playerA, "Yes"); // Pay replicate once
```

Comment:
you can actually use `setChoice(playerA, true)` here instead

Reply:
Is that preferred over "Yes"?

Reply:
yeah, there's less room for error that way

Reply:
> yeah, there's less room for error that way

Done